tests/test_utils.py

```diff
@@ -49,7 +49,7 @@ def test_overlapping(self):
     def test_overlapping_gpt2(self):
         raw = ["Indico Is the best hey"]
         finetunex = [
-            ["Indico ", "Is the", " best", " hey"]
+            ["Indico", " Is the", " best", " hey"]
```

Comment:
Is this consistent on gpt-2 that the spaces always occur at the beginning? Can we somehow force this consistency in gpt-1 without effecting anything else?